Package_SON.pretty/VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm.kicad_mod

```diff
@@ -0,0 +1,44 @@
+(module VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (layer F.Cu) (tedit 5B9F93A0)
+  (descr http://rohmfs.rohm.com/en/techdata_basic/ic/package/vson010v3030_1-e.pdf)
+  (tags "VSON-10 3x3mm")
+  (attr smd)
+  (fp_text reference REF** (at -0.006 2.572) (layer F.SilkS)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (at 0.248 -2.635) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -1.5 -1.5) (end -1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -1.5 1.5) (end 1.5 1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 1.5) (end 1.5 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 1.5 -1.5) (end -1.5 -1.5) (layer F.Fab) (width 0.1))
+  (fp_line (start -1.6764 -1.6764) (end -1.6764 1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.6764 1.6764) (end 1.6764 1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.6764 1.6764) (end 1.6764 -1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start 1.6764 -1.6764) (end -1.6764 -1.6764) (layer F.SilkS) (width 0.12))
+  (fp_line (start -0.8636 -1.8796) (end -1.8288 -1.8796) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.8796 -1.8796) (end -1.8796 -0.9652) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.78 -1.78) (end -1.78 1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.78 1.78) (end 1.78 1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.78 1.78) (end 1.78 -1.78) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.78 -1.78) (end -1.78 -1.78) (layer F.CrtYd) (width 0.05))
+  (fp_text user %R (at 0 0) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (pad 1 smd rect (at -1.4 -1) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 2 smd rect (at -1.4 -0.5) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 3 smd rect (at -1.4 0) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 4 smd rect (at -1.4 0.5) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 5 smd rect (at -1.4 1) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 6 smd rect (at 1.4 1) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 7 smd rect (at 1.4 0.5) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 8 smd rect (at 1.4 0) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 9 smd rect (at 1.4 -0.5) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 10 smd rect (at 1.4 -1) (size 0.8 0.27) (layers F.Cu F.Paste F.Mask))
+  (pad 11 smd rect (at 0 0) (size 1 2.1) (layers F.Cu F.Paste F.Mask))
+  (model ${KISYS3DMOD}/Package_SON.3dshapes/VSON-10_3.0x3.0mm_Pitch0.5mm_EP2x1.2.wrl
```

Comment:
The EP dimensions are incorrect, and in the wrong order. They should be: VSON-10-1EP_3x3mm_P0.5mm_EP**1x2.1mm**. This will require a change to the 3D model name in your [other PR](https://github.com/KiCad/kicad-packages3D/pull/401). Also, as above, consider adding the component height of 1mm.